locations/Bahraich/inputs/Optimisation/optimisation_inputs.yaml

```diff
@@ -0,0 +1,20 @@
+---
+################################################################################
+# optimisation_inputs.yaml - Specifications for running optimisations.         #
+#                                                                              #
+# Author: Phil Sandwell, Ben Winchester                                        #
+# Copyright: Phil Sandwell & Ben Winchester, 2021                              #
+# Date created: 14/07/2021                                                     #
+# License: Open source                                                         #
+################################################################################
+
```

Comment:
Is there (or do we need) a check that `iteration_length * number_of_iterations < 20`? I can't remember where we got to with this

Reply:
Ooh.. good point - I haven't got the optimisations working yet, so haven't found whether this causes a bug or not.. :tongue: . I'll leave this comment here as a note to remember to check it :smile: